pkg/codegen/docs/gen.go

```diff
@@ -1831,9 +1827,7 @@ func generateModulesFromSchemaPackage(tool string, pkg *schema.Package) map[stri
 	return modules
 }
 
-// GeneratePackage generates the docs package with docs for each resource given the Pulumi
-// schema.
-func GeneratePackage(tool string, pkg *schema.Package) (map[string][]byte, error) {
+func Initialize(tool string, pkg *schema.Package) {
```

Comment:
Adding this initialize allows to use the generated modules to do other things with it as separate operations rather a single function. It seemed like a good idea to decompose them into separate things.